package.json

```diff
@@ -16,14 +16,15 @@
     "@fortawesome/fontawesome-free": "^5.9.0",
     "bootstrap": "^4.3.1",
     "cross-env": "^6.0.3",
+    "dotenv-cli": "^4.0.0",
     "electron-devtools-installer": "^2.2.4",
     "electron-log": "^3.0.9",
```

Comment:
some of these deps should be moved to dev-dependencies (e.g. cross-env, dotenv-cli)